### DISCUSSION TITLE
What does `outputevent\itemid=226633` (labeled as 'Pre Admission') mean?

### DISCUSSION THREAD

**Discussion by arielhasidim** [MIT-LCP/mimic-code#1729] [MIMIC-IV] [answered]
1. Is this necessarily urine output?
2. Is this the first measurement of output in the ICU stay?
   - maybe a bag filled with previously unmeasured volume that the patient came in with? 
   - or maybe first catheter placement and drainage?
5. Or maybe it is a summation of outputs that were charted at the previous unit?

**Comment by alistairewj** [comment, marked as answer]
Usually it is the fluid output from the hospital stay, but before they were in the ICU (in the emergency department, the operating room, or on the floor). Since the ICU information system (MetaVision) automatically calculates fluid balance from all the fluid measurements, some nurses like to document the pre-admission fluid output in order to get a more accurate read of fluid balance.
